ansible/group_vars/sonic/vars

```diff
@@ -21,12 +21,13 @@ sensor_hwskus: [ "ACS-S6000", "ACS-MSN2700", "Force10-S6000" ]
 ## 2. https://groups.google.com/forum/#!msg/ansible-project/M-FNUK9Wz98/gH63Ka4hDAAJ
 
 syncd_docker_volumes_dict:
+  "/etc/ssw/:/etc/ssw/:ro":
```

Comment:
(Sorry, I should reply this thread)
Have you tested moving them inside container instead of in host? https://github.com/Azure/sonic-mgmt/blob/master/ansible/roles/sonicv2/tasks/quagga.yml has an example (see the 'block'). Roughly speaking, ansible can run any task directly in the container. 

We try to move ssw directory into specific containers in future. We don't want to add more complexity if we have workaround. In this case, BRCM, MLNX don't need the new mounting point.

Sorry for the inconvenience.